### PR TITLE
update dogstatsd docs

### PR DESCRIPTION
**Pull request by DaniCalifornia**
udpate dogstatsd docs with enc/docker origin detection setup
based on this escalation: https://datadoghq.atlassian.net/browse/FRCONS-624

### CODE REVIEW

File: content/en/developers/dogstatsd/_index.md [hunk 1]
```diff
@@ -96,7 +96,15 @@ docker run -d --cgroupns host \
...
-If you need to change the port used to collect StatsD metrics, use the `DD_DOGSTATSD_PORT="<NEW_DOGSTATSD_PORT>` environment variable. You can also configure DogStatsD to use a [Unix domain socket][1]:
+If you need to change the port used to collect StatsD metrics, use the `DD_DOGSTATSD_PORT="<NEW_DOGSTATSD_PORT>` environment variable. You can also configure DogStatsD to use a [Unix domain socket][1]
+
+#### Origin detection over UDP
+
+Origin detection is supported in Agent 6.10.0+ and allows DogStatsD to detect where the container metrics come from, and tag metrics automatically. When this mode is enabled, all metrics received through UDP are tagged by the same pod tags as Autodiscovery metrics.
+
+Origin detection in non-Kubernetes environments is based on an extension of the dogstatsd protocol: [Datagram Format and Shell Usage](https://docs.datadoghq.com/developers/dogstatsd/datagram_shell/?tab=metrics#dogstatsd-protocol-v12). To enable the feature in the agent the environment variable `DD_DOGSTATSD_ORIGIN_DETECTION_CLIENT=true` needs to be set.
+
+**Note:** Origin Detection is not supported for Fargate environments.
```
Comment (rtrieu) [inline content/en/developers/dogstatsd/_index.md:107]:
```suggestion
**Note**: Origin Detection is not supported for Fargate environments.
```

**Review by rtrieu — APPROVED** [review]
Approved, with minor feedback.

File: content/en/developers/dogstatsd/_index.md [hunk 2]
```diff
@@ -96,7 +96,15 @@ docker run -d --cgroupns host \
               gcr.io/datadoghq/agent:latest
 ```
 
-If you need to change the port used to collect StatsD metrics, use the `DD_DOGSTATSD_PORT="<NEW_DOGSTATSD_PORT>` environment variable. You can also configure DogStatsD to use a [Unix domain socket][1]:
+If you need to change the port used to collect StatsD metrics, use the `DD_DOGSTATSD_PORT="<NEW_DOGSTATSD_PORT>` environment variable. You can also configure DogStatsD to use a [Unix domain socket][1]
+
+#### Origin detection over UDP
+
+Origin detection is supported in Agent 6.10.0+ and allows DogStatsD to detect where the container metrics come from, and tag metrics automatically. When this mode is enabled, all metrics received through UDP are tagged by the same pod tags as Autodiscovery metrics.
```
Comment (alai97) [inline content/en/developers/dogstatsd/_index.md:103]:
```suggestion
Origin detection is supported in Agent v6.10.0+, and allows DogStatsD to detect where the container metrics come from and automatically tag metrics. When this mode is enabled, all metrics received through UDP are tagged by the same pod tags as Autodiscovery metrics.
```

File: content/en/developers/dogstatsd/_index.md [hunk 3]
```diff
@@ -96,7 +96,15 @@ docker run -d --cgroupns host \
               gcr.io/datadoghq/agent:latest
 ```
 
-If you need to change the port used to collect StatsD metrics, use the `DD_DOGSTATSD_PORT="<NEW_DOGSTATSD_PORT>` environment variable. You can also configure DogStatsD to use a [Unix domain socket][1]:
+If you need to change the port used to collect StatsD metrics, use the `DD_DOGSTATSD_PORT="<NEW_DOGSTATSD_PORT>` environment variable. You can also configure DogStatsD to use a [Unix domain socket][1]
```
Comment (alai97) [inline content/en/developers/dogstatsd/_index.md:99]:
```suggestion
If you need to change the port used to collect StatsD metrics, use the `DD_DOGSTATSD_PORT="<NEW_DOGSTATSD_PORT>` environment variable. You can also configure DogStatsD to use a [Unix domain socket][1].
```

File: content/en/developers/dogstatsd/_index.md [hunk 4]
```diff
@@ -96,7 +96,15 @@ docker run -d --cgroupns host \
...
+
+#### Origin detection over UDP
+
+Origin detection is supported in Agent 6.10.0+ and allows DogStatsD to detect where the container metrics come from, and tag metrics automatically. When this mode is enabled, all metrics received through UDP are tagged by the same pod tags as Autodiscovery metrics.
+
+Origin detection in non-Kubernetes environments is based on an extension of the dogstatsd protocol: [Datagram Format and Shell Usage](https://docs.datadoghq.com/developers/dogstatsd/datagram_shell/?tab=metrics#dogstatsd-protocol-v12). To enable the feature in the agent the environment variable `DD_DOGSTATSD_ORIGIN_DETECTION_CLIENT=true` needs to be set.
+
+**Note**: Origin Detection is not supported for Fargate environments.
 
 [1]: /developers/dogstatsd/unix_socket/
```
Comment (alai97) [inline content/en/developers/dogstatsd/_index.md:109]:
```suggestion
[1]: /developers/dogstatsd/unix_socket/
[2]: /developers/dogstatsd/datagram_shell/?tab=metrics#dogstatsd-protocol-v12
```

File: content/en/developers/dogstatsd/_index.md [hunk 5]
```diff
@@ -96,7 +96,15 @@ docker run -d --cgroupns host \
...
 ```
 
-If you need to change the port used to collect StatsD metrics, use the `DD_DOGSTATSD_PORT="<NEW_DOGSTATSD_PORT>` environment variable. You can also configure DogStatsD to use a [Unix domain socket][1]:
+If you need to change the port used to collect StatsD metrics, use the `DD_DOGSTATSD_PORT="<NEW_DOGSTATSD_PORT>` environment variable. You can also configure DogStatsD to use a [Unix domain socket][1]
+
+#### Origin detection over UDP
+
+Origin detection is supported in Agent 6.10.0+ and allows DogStatsD to detect where the container metrics come from, and tag metrics automatically. When this mode is enabled, all metrics received through UDP are tagged by the same pod tags as Autodiscovery metrics.
+
+Origin detection in non-Kubernetes environments is based on an extension of the dogstatsd protocol: [Datagram Format and Shell Usage](https://docs.datadoghq.com/developers/dogstatsd/datagram_shell/?tab=metrics#dogstatsd-protocol-v12). To enable the feature in the agent the environment variable `DD_DOGSTATSD_ORIGIN_DETECTION_CLIENT=true` needs to be set.
```
Comment (alai97) [inline content/en/developers/dogstatsd/_index.md:105]:
```suggestion
Origin detection in non-Kubernetes environments is based on an extension of the DogStatsD protocol in [Datagram Format and Shell Usage][2]. To enable the feature in the Agent, set the `DD_DOGSTATSD_ORIGIN_DETECTION_CLIENT` environment variable to `true`.
```

File: content/en/developers/dogstatsd/_index.md [hunk 6]
```diff
@@ -96,7 +96,15 @@ docker run -d --cgroupns host \
...
+
+**Note**: Origin Detection is not supported for Fargate environments.
```
Comment (alai97) [inline content/en/developers/dogstatsd/_index.md:107]:
```suggestion
**Note**: Origin detection is not supported for Fargate environments.
```

**Pull request merged**